.github/workflows/ci.yml

```diff
@@ -148,13 +148,18 @@ jobs:
         run: npm run test-types
       - name: Run all tests (for non-Linux)
         if: ${{ matrix.os != 'ubuntu-latest' }}
-        run: npm run test -- --test-suite ${{ matrix.suite }}
+        run: npm run test -- --test-suite ${{ matrix.suite }} --save-stats-to /tmp/artifacts/${{ github.event_name }}_${{ github.run_id }}_INSERTID.json
       - name: Install linux dependencies.
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: sudo apt-get install xvfb
       - name: Run all tests (for Linux)
         if: ${{ matrix.os == 'ubuntu-latest' }}
-        run: xvfb-run --auto-servernum npm run test -- --test-suite ${{ matrix.suite }}
+        run: xvfb-run --auto-servernum npm run test -- --test-suite ${{ matrix.suite }} --save-stats-to /tmp/artifacts/${{ github.event_name }}_${{ github.run_id }}_INSERTID.json
```

Comment:
Can you try with `${{ github.run_id }}_${{ github.job }}_${{ github.run_attempt }}` ? That way will not have to do some funky runtime updates.

Reply:
Honestly, I would prefer to rely on custom script here. The proposal is still not unique I believe because run_attempt is the same for every matrix job and so is run_id and job.

Reply:
I think we also don't need github.run_id The cli arguments we can improve after we find this useful.